src/disc/disc.py

```diff
@@ -3,15 +3,36 @@
 import os
 
 from constants import guild_id
-
+from dataclasses import dataclass 
 # ----------------------- #
 # api
 
 DISCORD_CATEGORY_ID = 4
-ADMINISTRATOR = 1 << 3
-VIEW_CHANNEL = 1 << 10
-
-async def discord_request(
+ADMINISTRATOR = 0b1000
+VIEW_CHANNEL = 0b0010_0000_0000
+
+@dataclass
+class User:
+    id: str
+    username: str
+    discriminator: str
+    global_name: str = None
+    avatar: str = None
```

Comment:
In python `|` refers to the Union type. Since the default for `global_name` is None, the actual type is either None or str.

```suggestion
    global_name: str | None = None
    avatar: str | None = None
```

Reply:
Change made.